releasetasks/templates/firefox/bb_update_verify.yml.tmpl

```diff
@@ -10,6 +10,9 @@
         - {{ stableSlugId(upstream_builder) }}
         {% endfor %}
         - {{ stableSlugId("release-{}-{}_updates".format(branch, product)) }}
+        {% if update_verify_requires_cdn_push is defined and update_verify_requires_cdn_push %}
```

Comment:
since uptake_monitoring itself assumes the other tasks in this `requires` block is done, should we have a if/else here? as in, if requires_cdn_push: add_uptake_monitor, else: add artifact_gen_and_updates tasks.